paper-webserver.tex

```diff
@@ -227,8 +235,8 @@ \subsection{Service Virtualization}
 \begin{figure}[!hb]
 \begin{center}
 \includegraphics[width=\textwidth]{images/docker}
-\end{center}
-\caption{The various Docker containers used in Spfy. Arrows represent the connections between different containers, and the entire platform can be recreated with a single command using its Docker-Compose definition.}
+\end{centerbut }
```

Comment:
?

Reply:
hmmm, not sure what I did there

Reply:
It should read 
"The Docker containers used in Spfy. Arrows represent the connections between different containers, and the entire platform can be recreated with a single command using its Docker-Compose definition."